engine/source/materials/mat_share/rotos4.F

```diff
@@ -0,0 +1,60 @@
+Copyright>        OpenRadioss
+Copyright>        Copyright (C) 1986-2023 Altair Engineering Inc.
+Copyright>
+Copyright>        This program is free software: you can redistribute it and/or modify
+Copyright>        it under the terms of the GNU Affero General Public License as published by
+Copyright>        the Free Software Foundation, either version 3 of the License, or
+Copyright>        (at your option) any later version.
+Copyright>
+Copyright>        This program is distributed in the hope that it will be useful,
+Copyright>        but WITHOUT ANY WARRANTY; without even the implied warranty of
+Copyright>        MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+Copyright>        GNU Affero General Public License for more details.
+Copyright>
+Copyright>        You should have received a copy of the GNU Affero General Public License
+Copyright>        along with this program.  If not, see <https://www.gnu.org/licenses/>.
+Copyright>
+Copyright>
+Copyright>        Commercial Alternative: Altair Radioss Software
+Copyright>
+Copyright>        As an alternative to this open-source version, Altair also offers Altair Radioss
+Copyright>        software under a commercial license.  Contact Altair to discuss further if the
+Copyright>        commercial version may interest you: https://www.altair.com/radioss/.
+Chd|====================================================================
+Chd|  ROTOS4                        source/materials/mat_share/rotos4.F
+Chd|-- called by -----------
+Chd|        USERMAT_SHELL                 source/materials/mat_share/usermat_shell.F
+Chd|-- calls ---------------
+Chd|====================================================================
+      SUBROUTINE ROTOS4(JFT,JLT,SIGXX,SIGYY,SIGXY,SIGYX,DIR,NEL)
+C-----------------------------------------------
+C   I m p l i c i t   T y p e s
+C-----------------------------------------------
+#include      "implicit_f.inc"
+C-----------------------------------------------
+C   D u m m y   A r g u m e n t s
+C-----------------------------------------------
+      INTEGER JFT, JLT,NEL
```

Comment:
INTENT are missing

Reply:
Hi @sebastienVilleneuve , this subroutine is unchanged, it was moved into an indepedent source file. I added array sizes. And now following you comment I also just added INTENT satatus. thank you